### ISSUE TITLE
Really slow to render when not using concise

### ISSUE THREAD

**Issue by ryanvanderpol** [xiewang/react-native-emoticons#6]
This is a great component, but on an iPhone 7, rendering this with `concise={false}` takes 5+ seconds.  On most Android devices it's even longer (10 seconds).  I assume this is because there is a bunch of processing going on with 1100+ emojis, but this lag basically makes this unusable for me.

**Comment by srameshr** [comment]
Yes, same for me. Rendering is really really slow. I had to remove the plugin.
@ryanvanderpol Did you find any solution for this?

**Comment by xiewang** [comment]
Sorry late for this. if you still on and have have time on this, try the latest version(@1.0.7) and add  a props to the components. The props is asyncRender, and set it true on android. Let me knew that if this solution is valid.
Thanks a lot

`<Emoticons`
		` onEmoticonPress={this._onEmoticonPress.bind(this)}`
		` onBackspacePress={this._onBackspacePress.bind(this)}`
		` show={this.state.showEmoticons}`
		 `concise={true}`
		` showHistoryBar={true}`
		 `showPlusBar={true}`
                 `asyncRender={true}`
    ` />
`

**Comment by srameshr** [comment]
@xiewang Thanks for the update! But i ended up using https://github.com/brendan-rius/react-native-emoji-keyboard/. It turned out to be super fast. But its only for iOS. I will use your library for android later this month and let you know.
In the meanwhile please see to this, that setting ```concise``` to true does not render all the emojis and basically turns out to be not useful at all.
